src/main/scala/com/goyeau/kubernetes/client/KubernetesClient.scala

```diff
@@ -30,6 +30,8 @@ object KubernetesClient {
     BlazeClientBuilder[F](ExecutionContext.global, Option(SslContexts.fromConfig(config))).resource
       .map(httpClient => apply(httpClient, config))
 
-  def apply[F[_]: ConcurrentEffect](config: F[KubeConfig]): Resource[F, KubernetesClient[F]] =
+  def apply[F[_]: ConcurrentEffect](
+    config: F[KubeConfig]
+  )(implicit cs: ContextShift[IO]): Resource[F, KubernetesClient[F]] =
```

Comment:
So can we now change that to `F[_]: ConcurrentEffect: ContextShift ` with your refactoring? Or the `IO` is still needed?

Reply:
`ContextShift` is already removed. We can use `Async` type class to deal with akka-http Futures.

Reply:
That is awesome 😃

Reply:
Yes, hopefully akka-http can be replaced by fs2 later.